samples/sync/src/main/res/layout/activity_main.xml

```diff
@@ -130,7 +130,7 @@
                 style="?android:attr/listSeparatorTextViewStyle"
                 android:text="@string/devices" />
 
-            <FragmentContainerView android:name="org.mozilla.samples.sync.DeviceFragment"
+            <androidx.fragment.app.FragmentContainerView android:name="org.mozilla.samples.sync.DeviceFragment"
```

Comment:
I was getting weird unrelated crashes when trying to build and run the sample app, and Google told me to change this, and it worked. Does this seem familiar to anyone else, and maybe indicate a problem with my local build env?

(I don't think it's related to the changes from this PR, because I was getting the same build issues on a clean checkout of a-c).